src/main/java/com/openlattice/chronicle/ChronicleStudyApi.java

```diff
@@ -59,13 +62,34 @@
      * @return The internal chronicle id for a device. It can be used to track a single device across resets, app uninstalls,
      * etc.
      */
+    @Deprecated( since = "apps v2" )
     @POST( BASE + STUDY_ID_PATH + PARTICIPANT_ID_PATH + DATASOURCE_ID_PATH )
     UUID enrollSource(
             @Path( STUDY_ID ) UUID studyId,
             @Path( PARTICIPANT_ID ) String participantId,
             @Path( DATASOURCE_ID ) String datasourceId,
             @Body Optional<Datasource> datasource );
 
+    /**
+     * Apps v2: Enrolls a participant's data datasource in a study.
```

Comment:
Minor. Shouldn't it be `participant's data source` instead of `participant's data datasource`?